frontend/admin/src/js/components/dashboard/Dashboard.tsx

```diff
@@ -169,34 +62,54 @@ export const Dashboard: React.FC<{
         })}
       </a>
 
-      <div className="container">
-        <section
-          className="dashboard"
-          data-h2-flex-grid="b(stretch, contained, flush, none)"
-        >
-          <div
-            data-h2-bg-color="b(lightnavy)"
-            data-h2-flex-item="b(1of1) m(1of4) l(1of6)"
-          >
-            <div
-              data-h2-padding="b(right-left, m)"
-              data-h2-position="b(static) m(sticky)"
-              style={{ top: "0", maxHeight: "100vh", overflow: "auto" }}
-            >
-              <SideMenu items={[...menuItems, LoginOrLogout()]} />
-            </div>
-          </div>
+      <div data-h2-display="b(flex)" data-h2-align-items="b(stretch)">
+        <SideMenu
+          isOpen={isMenuOpen}
+          onToggle={handleMenuToggle}
+          onDismiss={handleDismiss}
+        />
+        <SideMenuContentWrapper>
           <div
-            data-h2-flex-item="b(1of1) m(9of12) l(10of12)"
+            className="content-inner"
             data-h2-display="b(flex)"
-            style={{ flexDirection: "column" }}
+            data-h2-flex-direction="b(column)"
+            data-h2-align-items="b(space-between)"
           >
             <Header baseUrl={ADMIN_APP_DIR} />
-            <main id="main">{content}</main>
+            <main id="main" data-h2-margin="b(bottom, auto)">
+              {content}
+            </main>
             <Footer baseUrl={ADMIN_APP_DIR} />
           </div>
-        </section>
+        </SideMenuContentWrapper>
+      </div>
+      <div
+        data-h2-visibility="b(visible) m(hidden)"
+        data-h2-position="b(fixed)"
+        data-h2-location="b(bottom-right, xs)"
+        style={{ zIndex: 9998 }}
+      >
+        <Button
+          mode="solid"
+          color="secondary"
+          data-h2-display="b(inline-flex)"
+          data-h2-align-items="b(center)"
+          data-h2-shadow="b(s)"
+          onClick={handleMenuToggle}
+        >
+          <MenuIcon
+            style={{ width: "1rem", height: "1rem", marginRight: "0.5rem" }}
+          />
+          <span>
+            {intl.formatMessage({
+              defaultMessage: "Open Menu",
+              description: "Text label for header button that opens side menu.",
+            })}
+          </span>
+        </Button>
```

Comment:
Should this button be hidden when the menu is already open? It sticks around and the edge poking out from behind looks a little awkward.

<img width="1006" alt="Screen Shot 2022-05-17 at 4 53 02 PM" src="https://user-images.githubusercontent.com/305339/168908322-8884898a-e483-4a46-b079-08035b8b628e.png">

Reply:
Also, for legibility, I often tease apart smaller components, though I don't store them in their own files, nor even export them. So like a `isMenuOpen ? null : <MobileOpenMenuButton onClickHandler={handleMenuToggle} />` 

Thought admittedly, this is a little less cumbersome in regular javascript, where it just feels like a helper function and prop definitions don't make it seem so heavy.

Not sure if this is your vibe. I've taken that "Thinking in React" article to heart, and tend to create components for any visually distinct items (and hide the presentation styles inside them), but I don't bog the project down with a file for each, at least not until things are really settled.

Reply:
So, I have moved it to its own component to make the return statement more readable. However, hiding it becomes tricky for a11y since we are using a focus trap to assist users with focus management.

## Focus Trap

1. User opens menu
2. Focus shifts to the first focusable item in the menu and prevents the user from changing focus outside of the menu (traps them)
3. User closes the menu (either with `esc` or by clicking "Close Menu")
4. Focus trap is released and focus shifts back to the element that invoked the trap (the button)

### Problem

If we remove the button from the DOM or hide it, the focus trap no longer knows what initially invoked it so it can't put focus in the proper spot. So we are kind of stuck with the button staying on screen.

### Solution?

The exact placement, style etc. were all decisions I made (close to bottom of screen for short thumbs) since we were  unfortunately not provided this in the mockups. So we can open this for discussion on where to place it so that it is not in an awkward position no matter the menu state.

Reply:
> close to bottom of screen for short thumbs

strong agree. and thanks for adding it! I have a friend who uses an assistive input device (and consults related to it), and thinking more about moving buttons and reach lately

focus trap considerations are new to me, so I'll trust you on that. Would this be a reason to add an `isVisible` prop which just hides it but keep the element itself in the DOM?

Reply:
> focus trap considerations are new to me, so I'll trust you on that. Would this be a reason to add an isVisible prop which just hides it but keep the element itself in the DOM?

Yeah, i'll need to play around with that we can do to hide it will keeping it available to be focused later on.

Reply:
So, I've been trying different things and it seems opacity is the best way to do this. Applying different `data-h2-*` to try and use hyddrogen resulted in re-render for some reason that cause issues with the focus trap.

Updated with [hide toggle button when menu open](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2728/commits/7183c3df4f50b138d05741ce67389d66ebe85988)